games/views.py

```diff
@@ -19,5 +34,33 @@ def detail(request, id):
     context = {"games": IG.get_game_by_id(id)}
     twitter = {"twitter": tw.get_tweets(context["games"][0]["name"])}
     context |= twitter
-    # import pdb;pdb.set_trace()
     return render(request, "games/detail.html", context)
+
+
+def favorite(request):
+    """
+    Возвращаем игры с лайками
+    """
+    user = request.user.id
+    games = Game.objects.filter(likes=user)
+    context = {"games": games}
+    return render(request, "games/main.html", context)
+
+
+@login_required
+def like(request):
+    if request.method == "GET":
+        game_id = request.GET["game_id"]
+        likedgame = Game.objects.get(id=game_id)
```

Comment:
better to use `get_object_or_404` https://docs.djangoproject.com/en/4.0/topics/http/shortcuts/#get-object-or-404